src/openrct2/ride/RideData.h

```diff
@@ -276,6 +276,7 @@ enum ride_type_flags : uint64_t
     RIDE_TYPE_FLAG_HAS_LANDSCAPE_DOORS = (1ULL << 51),
     RIDE_TYPE_FLAG_UP_INCLINE_REQUIRES_LIFT = (1ULL << 52),
     RIDE_TYPE_FLAG_PEEP_CAN_USE_UMBRELLA = (1ULL << 53),
+    RIDE_TYPE_FLAG_HAS_CASH = (1ULL << 54),
```

Comment:
Has cash is a bit of an odd name. Is cash machine would bring it in alignment with Is toilet.

Reply:
I agree. Could you rename this flag to `RIDE_TYPE_FLAG_IS_CASH_MACHINE`?